cmd/cost/cost.go

```diff
@@ -82,13 +95,13 @@ func (opsCost *costOptions) initAWSClients() (awsprovider.Client, error) {
 	return awsClient, err
 }
 
-//Initializes costOptions struct
-func (opsCost *costOptions) complete(cmd *cobra.Command, _ []string) error {
-	if opsCost.accessKeyID == "" && opsCost.secretAccessKey == "" {
-		fmt.Fprintln(opsCost.Out, "AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are not provided, reading credentials from config file or env vars.")
-	} else if opsCost.accessKeyID == "" || opsCost.secretAccessKey == "" {
-		return cmdutil.UsageErrorf(cmd, "The flag aws-access-key-id and aws-secret-access-key should be set or not set at the same time")
+func getOU(org awsprovider.Client, OUid string) *organizations.OrganizationalUnit {
+	result, err := org.DescribeOrganizationalUnit(&organizations.DescribeOrganizationalUnitInput{
+		OrganizationalUnitId: aws.String(OUid),
+	})
```

Comment:
what happens if `result` return empty? eg. given an OU that does not have any child OUs, does this hit an error or will it return an empty list?

Reply:
The way I understand DescribeOrganizationalUnit is that it returns information regarding an OU. As long as the OUid given for the field corresponds to an existing OU, there won't be an error. If there doesn't exist an OU with the OUid, there will be an "InvalidInputException: You provided a value that does not match the required pattern" error.

If the OU doesn't have any child OUs, the information for the OU would still be returned to result. Later on, the different commands (get, reconcile) will not loop through the child OUs of the chosen OU, as there aren't any to loop through.

Reply:
Does this function call operate on child OUs? It seems to only return the name of the OU specified.

Reply:
Sorry I wasn't clear--it does not operate on child OUs.

Reply:
:+1: Yep, looks good to me then. @sam-nguyen7 I don't think we have to worry about child OUs here